src/promptflow/tests/sdk_cli_azure_test/conftest.py

```diff
@@ -126,3 +158,9 @@ def ml_client_canary(
         workspace_name="promptflow-canary-dev",
         cloud="AzureCloud",
     )
+
+
+@pytest.fixture(autouse=True)
+def mock_appinsights_log_handler(mocker: MockFixture) -> None:
+    dummy_logger = logging.getLogger("dummy")
+    mocker.patch("promptflow._telemetry.telemetry.get_telemetry_logger", return_value=dummy_logger)
```

Comment:
Why add this? I thought test logs won't get into telemetry in the first place?

Reply:
this fixture targets to mock the init operation, which will generate some requests with sensitive data